### PR TITLE
WG-Batch leads update

### PR DESCRIPTION
**Pull request by mwielgus**
@ahg-g decided to step down from the chair role and will be succeed by @mwielgus.

### CODE REVIEW

File: wg-batch/README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ The [charter](charter.md) defines the scope and governance of the Batch Working
 ## Organizers
 
 * Wei Huang (**[@Huang-Wei](https://github.com/Huang-Wei)**), Apple
-* Abdullah Gharaibeh (**[@ahg-g](https://github.com/ahg-g)**), Google
 * Danielle Lancashire (**[@endocrimes](https://github.com/endocrimes)**), Independent
+* Marcin Wielgus (**[@mwielgus](https://github.com/mwielgus)**), Google
 * Maciej Szulik (**[@soltysh](https://github.com/soltysh)**), Red Hat
 * Swati Sehgal (**[@swatisehgal](https://github.com/swatisehgal)**), Red Hat
```
[the remark this answers is not in the document]
Comment (mwielgus) [reply wg-batch/README.md:34]: sure

[the remark this answers is not in the document]
Comment (mwielgus) [reply wg-batch/README.md:34]: Done.

**Comment by cblecker** [comment]
@mwielgus has this leadership change followed the process outlined here?: https://git.k8s.io/community/contributors/chairs-and-techleads/leadership-changes.md

**Comment by ahg-g** [comment]
> @mwielgus has this leadership change followed the process outlined here?: https://git.k8s.io/community/contributors/chairs-and-techleads/leadership-changes.md

I think so, pls see this thread: https://groups.google.com/a/kubernetes.io/g/wg-batch/c/IXWHR-jkoRM

**Review by endocrimes — APPROVED** [review]
/lgtm

**Review by soltysh — APPROVED** [review]
/lgtm
/hold
for @mrbobbytables final approval, since he's the liaison for wg-batch

**Comment by mrbobbytables** [comment]
/lgtm
/hold cancel

remember to follow up with the leadership change issue template to make sure everything is updated in all the relevant places :+1:

**Pull request merged**